public/app/stores/gateway/gateway-store.js

```diff
@@ -9,24 +11,45 @@ module.exports = Reflux.createStore({
     this.onFetchGateway();
   },
 
+  getSetupAppUrl: function getSetupAppUrl () {
+
+    function isOSX () {
+      var userAgentString = navigator.userAgent;
+      var environment = platform.parse(userAgentString);
+      return /OS X/.test(environment.os.family);
+    }
+
+    if (isOSX(platform)) {
+      return window.yetu.config.setupDownloadUrlMac;
+    } else {
```

Comment:
Shouldn't better inform other OS users that their platform isn't supported? Otherwise Linux users might guess that they are just presented the wrong download URL.